drivers/CoreVideo/external/source/HAL_IRQHandlers.cpp

```diff
@@ -0,0 +1,34 @@
+#include "CoreVideo.hpp"
+
+extern "C" {
+
+namespace display::internal {
+	extern leka::CoreVideo corevideo;
+}
```

Comment:
to continue from: https://github.com/leka/LekaOS/pull/1172#discussion_r1042361642

if I don't use a namespace here, I cannot have 

```
extern "C" auto corevideo =
		CoreVideo {hal, coresdram, coredma2d, coredsi, coreltdc, corelcd, coregraphics, corefont, corejpeg};
```

in a namespace, because the linker won't be able to find it.

the definition and declaration must have the same namespace.

as we had `display::internal::corevideo` in os/main.cpp, it was easier to follow the convention than to change it.

again, I'm not sure it's the issue here, I'm happy to discuss the formatting of the namespace structures in another PR if needed.

changes in the spikes we not made "because I wanted to" but because of these lines as well. Using the same namespace structure, as explained before, felt like the good way to do it.